### DISCUSSION TITLE
"Allow only one instance to run" option and "/x" command line parameter

### DISCUSSION THREAD

**Discussion by goozak** [WinMerge/winmerge#504] [Q&A] [answered]
Wow...  This place is brand new!  Hello!  👋  I hope I'm at the right place for this...

I'm using the "Allow only one instance to run" to prevent having multiple copies of WinMerge strewn about on my desktop — I tend to lose track of them otherwise.

But if I use WinMerge thru the command line with the "/x" parameter, I _really_ don't want it to mess with my opened windows...

Is there a way to do a comparison and close if equal without disturbing an already opened window?  I feel like this should be the default behaviour, regardless of the "Allow only one..." option.

Or maybe something like a "/s-" parameter (notice the 'minus') could allow WinMerge to override the "Allow only one..." option and launch in its own window and do its business, including closing, without interfering with existing windows?

**Comment by sdottaka** [comment, marked as answer]
> Or maybe something like a "/s-" parameter (notice the 'minus') could allow WinMerge to override the "Allow only one..." option and launch in its own window and do its business, including closing, without interfering with existing windows?

It's not in the manual, but I think the following command will work.

"c:\Program Files\WinMerge\WinMergeU.exe" -cfg SingleInstance=0 path1 path2

**Reply by goozak** [reply]
Exactly what I was looking for!  Thanks 😄

**Reply by sdottaka** [reply]
In WinMerge 2.16.10, the /s- command line option has been added.
